compose.example.yml

```diff
@@ -0,0 +1,15 @@
+---
+services:
+  sml2mqtt:
+    image: spacemanspiff2007/sml2mqtt
+    container_name: sml2mqtt
+    init: true
+    mem_limit: 32m
+    restart: unless-stopped
+    environment:
+      USER_ID: 1000
+      GROUP_ID: 20
+    volumes:
+      - "./config.yml:/sml2mqtt/config.yml"
```

Comment:
The example should mount the volume because it also contains a possible logfile

Reply:
Done.